daycare/models.py

```diff
@@ -39,3 +47,24 @@ def create(email, username, password, name, description, price_per_day, capacity
         new_daycare.save()
 
         return new_daycare
+
+
+class Image(models.Model):
+    name = models.CharField(max_length=255)
+    image = models.CharField(max_length=1000)
+    daycare_id = models.ForeignKey(DayCare, null=True, on_delete=models.SET_NULL)
+
+    @classmethod
+    def create(cls, name, image, daycare_id):
+
+        new_image = Image(name=name, image=image, daycare_id=daycare_id)
```

Comment:
what about validations? for example 
- the length of the URL
- DayCare exist

Reply:
Done